tests/Integration/Routing/ImplicitBindingTest.php

```diff
@@ -23,32 +22,44 @@ protected function getEnvironmentSetUp($app)
         ]);
     }
 
-    protected function setUp(): void
+    protected function defineDatabaseMigrations(): void
     {
-        parent::setUp();
-
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
             $table->timestamps();
         });
+
+        $this->beforeApplicationDestroyed(function () {
+            Schema::dropIfExists('users');
+        });
```

Comment:
Need to drop schema since we going to reload the application during tests (after `route:cache`).